modules/ROOT/pages/dw-runtime-types.adoc

```diff
@@ -1,12 +1,146 @@
 = Runtime Types (dw::Runtime)
 
-
 |===
 | Type | Definition | Description
 
+| DataFormatDescriptor
```

Comment:
TODO: MISSING descriptions and annotation or expansion to a notification that the feature is experimental